apps/pearl-api/context/Web3AuthProvider.tsx

```diff
@@ -5,12 +5,15 @@ import {
 } from '@web3auth/modal/react';
 import { PropsWithChildren } from 'react';
 
-const clientId = `${process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID}`;
+import { CHAIN_CONFIGS } from '../utils/web3auth';
+
+// const clientId = `${process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID}`;
+const clientId = `${process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID_STAGING}`; // TODO: replace with above one
 
 const web3AuthContextConfig: Web3AuthContextConfig = {
   web3AuthOptions: {
     clientId,
-    web3AuthNetwork: WEB3AUTH_NETWORK.SAPPHIRE_MAINNET,
+    web3AuthNetwork: WEB3AUTH_NETWORK.SAPPHIRE_DEVNET,
```

Comment:
maybe we can use [VERCEL_ENV](https://vercel.com/docs/environment-variables/system-environment-variables#VERCEL_ENV) to define the network, not to change it back before merging to prod?